source/cloud-security/amazon/services/supported-services/security-lake.rst

```diff
@@ -173,8 +173,11 @@ Queue configuration
 Authentication
 ^^^^^^^^^^^^^^
 
+The available authentication configuration parameters are the following:
+
```

Comment:
```suggestion
```

Reply:
It's not originally  requested in the associated issue and it's redundant due to [line 165](https://github.com/wazuh/wazuh-documentation/pull/8199/files#diff-651036a831d9e22f1ea5144d53f9876a76928e6e9c28c1959922c1673fcea61cR165) above and not consistent with`Queue configuration` above.